extensions/ql-vscode/src/interface-utils.ts

```diff
@@ -116,11 +117,38 @@ export function tryResolveLocation(
  * Uses a content security policy that only loads the given script.
  */
 export function getHtmlForWebview(
+  ctx: ExtensionContext,
   webview: Webview,
-  scriptUriOnDisk: Uri,
-  stylesheetUrisOnDisk: Uri[],
-  allowInlineStyles: boolean
+  view: 'results' | 'compare' | 'remote-queries',
+  {
+    allowInlineStyles,
+    includeCodicons
+  }: {
+    allowInlineStyles?: boolean;
+    includeCodicons?: boolean;
+  } = {
+      allowInlineStyles: false,
```

Comment:
I believe this is only `true` for the mrva page so that it can use inline styles while loading react components from the primer UI toolkit library. Since we are moving away from primer and using the vscode UI toolkit, is this still necessary?

Again, not related to this PR, but perhaps something to check later.